themes/uhsg_theme/uhsg_theme.theme

```diff
@@ -435,3 +430,9 @@ function uhsg_theme_theme_suggestions_field_alter(&$suggestions, $vars, $hook) {
     $suggestions[] = 'field__tag_list_item';
   }
 }
+
+function _uhsg_theme_get_degree_programme_switcher() {
+  $degree_programme_switcher = \Drupal\block\Entity\Block::load('views_block__degree_programmes_block_1');
+
+  return \Drupal::entityTypeManager()->getViewBuilder('block')->view($degree_programme_switcher);
```

Comment:
This block can be removed from UI therefore in code we should prepare for not finding this block. `view` method expects here an `EntityInterface` and `Block::load` might return `null` instead, therefore there is a change that fatal error is thrown here if somebody happens to remove, or rename the block.